src/app/components/login-card/login-card.component.ts

```diff
@@ -1,15 +1,87 @@
 import { Component, OnInit } from '@angular/core';
+import { FormBuilder, FormControl, FormGroup, Validators } from '@angular/forms';
+import { MatSnackBar } from '@angular/material/snack-bar';
+import { Router } from '@angular/router';
+import { loginUser } from 'src/app/models/loginUser';
+import { User } from 'src/app/models/user';
+import { AuthenticationService } from 'src/app/services/auth.service';
 
 @Component({
   selector: 'app-login-card',
   templateUrl: './login-card.component.html',
-  styleUrls: ['./login-card.component.scss']
+  styleUrls: ['./login-card.component.scss'],
 })
 export class LoginCardComponent implements OnInit {
+  loginForm!: FormGroup;
+  isInputValid: boolean = false;
+  loginStatus: boolean = false;
+  loginMessage: string = '';
+  error = '';
 
-  constructor() { }
+  responseUserObject?: loginUser;
+
+  constructor(
+    private formBuilder: FormBuilder,
+    private loginService: AuthenticationService,
+    private snackBar: MatSnackBar,
+    private router: Router
+  ) {if (this.loginService.currentUserValue) { 
+            this.router.navigate(['/events-list']);
+  }
+  }
 
   ngOnInit(): void {
+    debugger;
+    this.loginForm = this.formBuilder.group({
+      loginEmailControl: new FormControl('', [
+        Validators.required,
+        Validators.pattern('[a-zA-Z]+\\.[a-zA-Z]+@tss-yonder\\.com'),
+      ]),
+      loginPasswordControl: new FormControl('', [
+        Validators.required,
+        Validators.minLength(6),
+      ])
+    });
+  }
+
+  isLoginFormCompleted(): boolean {
+    return this.loginForm.touched;
   }
 
+  isLoginFormValid(): boolean {
+    return this.loginForm.valid;
+  }
+
+  isLoginFormDone(): boolean {
+    return this.isLoginFormCompleted() && this.isLoginFormValid();
+  }
+
+  onLoginUser(): void {
+    debugger;
```

Comment:
remove the debugger; search for occurrences in all the app, you have quite some in here